README.rst

```diff
@@ -910,6 +910,48 @@ List locations with devices deployed (in GeoJSON format)
 
     GET /api/v1/controller/location/geojson/
 
+List devices with location
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: text
+
+    GET /api/v1/controller/devicelocation/
```

Comment:
as already explained, this should be:

`/api/v1/controller/device/{id}/location/`

we have to improve this view:

https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/geo/api/views.py#L32-L65

editing the data should keep working with the current way, adding/deleting/editing can be done also with the user token.